pkg/cmd/server/origin/master_config.go

```diff
@@ -141,10 +151,25 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	admissionControlPluginNames := []string{"OriginNamespaceLifecycle"}
 	admissionController := admission.NewFromPlugins(privilegedLoopbackKubeClient, admissionControlPluginNames, "")
 
+	var tokenGetter serviceaccount.ServiceAccountTokenGetter
+	if options.KubernetesMasterConfig == nil {
```

Comment:
That doesn't seem likely.  It'd be impossible to bootstrap a system that way since the system has to be up to create the service account token you needed to validated.